tools/clang/test/DXC/Passes/PIX/debugi25.ll

```diff
@@ -0,0 +1,103 @@
+; RUN: %dxopt %s -dxil-annotate-with-virtual-regs -hlsl-dxil-debug-instrumentation -S | FileCheck %s
+
+; Expect an i25 cast, or this test isn't testing anything:
+; CHECK: [[CAST:%.*]] = trunc i32 %{{.*}} to i25
+;
+; Check that we correctly z-extended that i25 before trying to write it to i32
+; [[ZEXT:%.*]] = zext i25 [[CAST]] to i32
+; call void @dx.op.bufferStore.i32(i32 69, %dx.types.Handle %PIX_DebugUAV_Handle, i32 %{{.*}}, i32 undef, i32 [[ZEXT]]
```

Comment:
These lines are missing `CHECK:` prefix, so they won't be checked.

Reply:
D'Oh! Musta deleted them when copying the file. Still passes with them :-)